config/os.go

```diff
@@ -317,6 +317,7 @@ func GetEOL(family, release string) (eol EOL, found bool) {
 			"35": {StandardSupportUntil: time.Date(2022, 12, 12, 23, 59, 59, 0, time.UTC)},
 			"36": {StandardSupportUntil: time.Date(2023, 5, 16, 23, 59, 59, 0, time.UTC)},
 			"37": {StandardSupportUntil: time.Date(2023, 12, 15, 23, 59, 59, 0, time.UTC)},
+			"38": {StandardSupportUntil: time.Date(2024, 5, 18, 23, 59, 59, 0, time.UTC)},
```

Comment:
Here the EOL is described as `2024-05-14`.
https://fedorapeople.org/groups/schedule/f-38/f-38-key-tasks.html

Reply:
Updated based on that link!
(btw, Fedora N's exact EOL date is not determined until N+2 is released)